src/backend/joanie/core/utils/payment_schedule.py

```diff
@@ -86,7 +86,7 @@ def _calculate_installments(total, due_dates, percentages):
     """
     Calculate the installments for the order.
     """
-    total_amount = Money(total, settings.DEFAULT_CURRENCY)
+    total_amount = Money(total)
```

Comment:
Why this change ?

Reply:
Some tests were failing until I use `order.refresh_from_db`

Actually, I think the currency was set but then Django stores value in database, this information was removed. Then when it is time to [serialize this value through a DecimalField](https://github.com/openfun/joanie/blob/9092bfd503793ad03f59a6e3614e71fba9d4d660/src/backend/joanie/core/serializers/client.py#L1040-L1046) an error was raised as according to the Money object has a currency set or not, its string representation is not the same and in case there is a currency set, the DecimalField raises an error.

```python
# Not ok to be serialized through a DecimalField
str(Money(100.00, settings.DEFAULT_CURRENCY)) # -> 159.00 €

# Ok to be serialized through a DecimalField
str(Money(100.00, settings.DEFAULT_CURRENCY)) # -> 159.00
```

Reply:
I had to do the same as @jbpenrath on my side. When I add to set the value as a `Money` object, I had to add also the `DEFAULT_CURRENCY`.